selfdrive/car/hyundai/values.py

```diff
@@ -1156,21 +1159,29 @@ class Buttons:
       b'\xf1\x00BD  MDPS C 1.00 1.02 56310-XX000 4BD2C102',
       b'\xf1\x00BD  MDPS C 1.00 1.08 56310/M6300 4BDDC108',
       b'\xf1\x00BD  MDPS C 1.00 1.08 56310M6300\x00 4BDDC108',
+      b'\xf1\x00BDm MDPS C A.01 1.03 56310M7800\x00 4BPMC103',
+      b'\xf1\x8756310M7800\x00\xf1\x00BDm MDPS C A.01 1.03 56310M7800\x00 4BPMC103',
```

Comment:
You only need to add the shortest response (usually), these are just responses from two slightly different FW queries. Perhaps we should add some brand-specific infrastructure to deal with cars that respond to both.

Reply:
Gotcha, that would clear up some confusions for new users. I will remove the longer response in the meantime.